docs/faq/tools.md

```diff
@@ -31,7 +31,13 @@ See the [installation guide](intro/install).
 Currently the main way to author a schema is to edit schema YAML files
 in a text editor or IDE (Integrated Development Environment).
 
-We recommend using an IDE that has support for YAML format
+We recommend using an IDE that has support for YAML format.
```

Comment:
Would be good to have a link to the LinkML json schema here which can be used for validating and auto-completion of fields in the yaml

Reply:
You can use https://w3id.org/linkml/meta.schema.json

This redirects to linkml_model/jsonschema/meta.schema.json in GitHub